### PR TITLE
update devcontainer

### PR DESCRIPTION
**Pull request by tonyfettes**

### CODE REVIEW

File: .devcontainer/moonbit/devcontainer-feature.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "moonbit",
+  "version": "0.0.1",
+  "name": "moonbitg",
+  "installsAfter": [
+    "ghcr.io/devcontainers/features/common-utils"
+  ]
+}
```
Comment (peter-jerry-ye) [inline .devcontainer/moonbit/devcontainer-feature.json:8]: What is this for?

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -3,11 +3,16 @@
 {
 	"name": "Ubuntu",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/base:jammy",
+	"build": {
+		"dockerfile": "dockerfile",
+		"options": [
+			"--platform=linux/amd64"
+		]
+	},
```
Comment (peter-jerry-ye) [inline .devcontainer/devcontainer.json:11]: Does this solve the problem running on M1? The original image was already built with linux/amd64

File: .devcontainer/moonbit/devcontainer-feature.json (hunk 1 above)
Comment (tonyfettes) [reply .devcontainer/moonbit/devcontainer-feature.json:8]: image

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (tonyfettes) [reply .devcontainer/devcontainer.json:11]: https://stackoverflow.com/a/78827072/8323399

**Review by peter-jerry-ye — APPROVED** [review]

**Pull request merged**